readthedocs/api/v2/serializers.py

```diff
@@ -188,16 +188,30 @@ class RemoteRepositorySerializer(serializers.ModelSerializer):
 
     # This field does create an additional query per object returned
     matches = serializers.SerializerMethodField()
+    admin = serializers.SerializerMethodField('is_admin')
 
     class Meta:
         model = RemoteRepository
-        exclude = ('json', 'users')
+        exclude = ('users',)
 
     def get_matches(self, obj):
         request = self.context['request']
         if request.user is not None and request.user.is_authenticated:
             return obj.matches(request.user)
 
+    def is_admin(self, obj):
+        request = self.context['request']
+
+        # Use cached value
```

Comment:
Where is this cache set?

Reply:
We discussed about this here https://github.com/readthedocs/readthedocs.org/pull/7778#discussion_r549993292

Reply:
Ha, we should add a comment then, if we both had the same questions :)